### PR TITLE
feat: adds metadataviews support to pack NFTs

### PR DESCRIPTION
**Pull request by KhaledAylii**
- Implements the metadata views standard to the NFL All Day PackNFT contract

References:
- [Metadata views official contract](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc)
- [Metadata views FLIP issue](https://github.com/onflow/flow/blob/master/flips/20210916-nft-metadata.md)

### CODE REVIEW

File: cadence-contracts/MetadataViews.cdc [hunk 1]
```diff
@@ -0,0 +1,739 @@
+import FungibleToken from "./FungibleToken.cdc"
```
Comment (KhaledAylii) [inline cadence-contracts/MetadataViews.cdc:1]: standard metadata views contract from official flow repo, can skip

File: flow.json [hunk 2]
```diff
@@ -41,16 +41,10 @@
   },
```
Comment (KhaledAylii) [inline flow.json:43]: auto linting, can skip

File: cadence-contracts/PackNFT.cdc [hunk 3]
```diff
@@ -129,13 +130,110 @@ pub contract PackNFT: NonFungibleToken, IPackNFT {
...
+        pub fun resolveView(_ view: Type): AnyStruct? {
+            switch view {
+                case Type<MetadataViews.Display>():
+                    return MetadataViews.Display(
+                        name: "NFL All Day Pack",
+                        description: "Reveals official NFL All Day Moments when opened",
+                        thumbnail: MetadataViews.HTTPFile(url: self.getImage(imageType: "image", format: "jpeg", width: 256))
+                    )
+                case Type<MetadataViews.ExternalURL>():
+                    return MetadataViews.ExternalURL("https://nflallday.com/packNFTs/".concat(self.id.toString())) // might have to make a URL that redirects to packs page based on packNFT id -> distribution id
```
Comment (KhaledAylii) [inline cadence-contracts/PackNFT.cdc]: need to implement this page that takes a packNFT's flow id and directs a user to the pack's distribution's page

**Review by EricLin-wk — APPROVED** [review]
LGTM!

**Pull request merged**